db/migrate/20150219173821_create_module_runs.rb

```diff
@@ -0,0 +1,21 @@
+class CreateModuleRuns < ActiveRecord::Migration
+  def change
+    create_table :module_runs do |t|
```

Comment:
`t.references` should be favored over `t.integer` for foreign keys to distinguish foreign keys from actual integer columns.